Makefile

```diff
@@ -1,4 +1,10 @@
 OPERATOR_LOGGING_IMAGE_STREAM?=stable
+
+FLUENTD_IMAGE?="openshift/origin-logging-fluentd"
+KIBANA_IMAGE?="openshift/origin-logging-kibana6"
+ELASTICSEARCH_IMAGE?="openshift/origin-logging-elasticsearch6"
+CURATOR_IMAGE?="openshift/origin-logging-curator5"
+
 # Build the Docker images for Origin Aggregated Logging
 build-images:
```

Comment:
i'm not sure if this is used by CI anywhere, need to investigate that further before removing it